app/views/hosts/_list.html.erb

```diff
@@ -19,6 +22,13 @@
         <td class="ca">
           <%= check_box_tag "host_ids[]", nil, false, :id => "host_ids_#{host.id}", :class => 'host_select_boxes', :onclick => 'hostChecked(this)' %>
         </td>
+        <% if power_status_visible? %>
+          <td class="power-status">
+            <span <%= vm_power_class(false) %>>
+              <%= _('Off') %>
```

Comment:
defaulting to an `Off` label when waiting for the AJAX is misleading, better to have a spinner.